### PR TITLE
[zh-cn]: create the translation of `Permission-Policy.bluetooth` directive

### PR DESCRIPTION
**Pull request by tianyeeT**
…ctive

### CODE REVIEW

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+slug: Web/HTTP/Headers/Permissions-Policy/bluetooth
+l10n:
+  sourceCommit: 0880a90f3811475d78bc4b2c344eb4146f25f66c
+---
+
+{{HTTPSidebar}} {{SeeCompatTable}}
+
+HTTP {{HTTPHeader("Permissions-Policy")}} 标头的 `bluetooth` 指令控制当前文档是否被允许使用 [Web Bluetooth API](/zh-CN/docs/Web/API/Web_Bluetooth_API)。
+
+具体来说，如果定义的策略不允许使用此功能，则由 {{domxref('Navigator.bluetooth')}} 返回的 {{domxref('Bluetooth')}} 对象的方法将会阻止访问：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:12]: feature => 特性。其他的几个 PR 也请麻烦改一下

Comment (tianyeeT) [reply files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:12]: 收到

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## 默认策略
+
+`bluetooth` 的默认允许列表为：`self`。
+
+## 示例
+
+### 通用示例
+
+SecureCorp Inc. 希望在所有浏览上下文中禁用 Web Bluetooth API，除了其自身的来源和 `https://example.com` 的来源。
+它可以通过发送以下 HTTP 响应头来定义权限策略：
```
Comment (PassionPenguin) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:36]:
```suggestion
SecureCorp Inc. 希望在所有浏览上下文中禁用 Web Bluetooth API，除了其自身的来源和 `https://example.com` 的来源。它可以通过发送以下 HTTP 响应头来定义权限策略：
```

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+```http
+Permissions-Policy: bluetooth=(self "https://example.com")
+```
+
+### 带有 \<iframe> 元素
+
+FastCorp Inc. 希望为所有跨来源的子框架禁用 `bluetooth`，除了特定的 `<iframe>`。
+它可以通过发送以下 HTTP 响应头来定义权限策略：
```
Comment (PassionPenguin) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:45]:
```suggestion
FastCorp Inc. 希望为除了特定的 `<iframe>` 的所有跨来源的子框架禁用 `bluetooth`。它可以通过发送以下 HTTP 响应头来定义权限策略：
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]

**Review by yin1999 — CHANGES_REQUESTED** [review]
https://github.com/mdn/translated-content/pull/22053#pullrequestreview-2147208712

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- {{HTTPHeader("Permissions-Policy")}} 标头
+- [Permissions Policy](/zh-CN/docs/Web/HTTP/Permissions_Policy)
```
Comment (skyclouds2001) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:70]:
```suggestion
- [权限策略](/zh-CN/docs/Web/HTTP/Permissions_Policy)
```

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+---
+title: "Permissions-Policy: bluetooth"
```
Comment (skyclouds2001) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:2]: 参考 https://github.com/mdn/translated-content/pull/22049#discussion_r1661735586

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md (hunk 1 above)
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:12]:
```suggestion
具体来说，如果定义的策略不允许使用此特性，则由 {{domxref('Navigator.bluetooth')}} 返回的 {{domxref('Bluetooth')}} 对象的方法将会阻止访问：
```

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+- {{DOMxRef("Bluetooth.requestDevice()")}} 将以 `SecurityError` 类型的 {{domxref("DOMException")}} 拒绝其返回的 {{jsxref("Promise")}}。
+
+## 语法
+
+```http
+Permissions-Policy: bluetooth=<allowlist>;
+```
+
+- `<allowlist>`
+  - : 允许使用该功能的来源列表。请参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)了解更多详细信息。
```
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:25]:
```suggestion
  - : 允许使用该特性的来源列表。请参阅 [`Permissions-Policy` > 语法](/zh-CN/docs/Web/HTTP/Headers/Permissions-Policy#语法)了解更多详细信息。
```

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+Permissions-Policy: bluetooth=(self https://other.com/blue)
+```
+
+然后在 `<iframe>` 元素上包含一个 {{HTMLElement('iframe','allow','#Attributes')}} 属性：
+
+```html
+<iframe src="https://other.com/blue" allow="bluetooth"></iframe>
+```
+
+`<iframe>` 属性可以有选择地在某些框架中启用功能，而在其他框架中禁用这些功能，即使这些框架包含来自同一来源的文档。
```
Comment (tianyeeT) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:57]:
```suggestion
`<iframe>` 属性可以有选择地在某些框架中启用特性，而在其他框架中禁用这些特性，即使这些框架包含来自同一来源的文档。
```

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+## 默认策略
+
+`bluetooth` 的默认允许列表为：`self`。
+
+## 示例
+
+### 通用示例
+
+SecureCorp Inc. 希望在所有浏览上下文中禁用 Web Bluetooth API，除了其自身的来源和 `https://example.com` 的来源。它可以通过发送以下 HTTP 响应头来定义权限策略：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:35]:
```suggestion
SecureCorp Inc. 希望在所有浏览上下文中禁用 Web Bluetooth API，除了其自身的来源和来源 `https://example.com`。它可以通过发送以下 HTTP 响应标头来定义权限策略：
```

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+```http
+Permissions-Policy: bluetooth=(self "https://example.com")
+```
+
+### 带有 \<iframe> 元素
+
+FastCorp Inc. 希望为除了特定的 `<iframe>` 的所有跨来源的子框架禁用 `bluetooth`。它可以通过发送以下 HTTP 响应头来定义权限策略：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:43]:
```suggestion
FastCorp Inc. 希望为除了特定的 `<iframe>` 的所有跨来源的子框架禁用 `bluetooth`。它可以通过发送以下 HTTP 响应标头来定义权限策略：
```

File: files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+
+```http
+Permissions-Policy: bluetooth=(self https://other.com/blue)
+```
+
+然后在 `<iframe>` 元素上包含一个 {{HTMLElement('iframe','allow','#Attributes')}} 属性：
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/permissions-policy/bluetooth/index.md:49]:
```suggestion
然后在 `<iframe>` 元素上包含一个 {{HTMLElement('iframe','allow','#属性')}} 属性：
```

**Review by yin1999 — COMMENTED** [review]
nits:

**Review by yin1999 — APPROVED** [review]

**Pull request merged**